### ISSUE TITLE
Install firmware from the SD card of the emulator

### ISSUE THREAD

**Issue by fruit-bat** [fruit-bat/pico-zxspectrum#86]
Raised on behalf of @digitalmexer.

I really don't know how but it is an interesting idea.

**Comment by ArnoldUK** [comment]
Yeah, it's possible but much quicker to simply plug the pico into your usb port and drag it over. Either way it's going to involve copying the firmware manually.
